backend/src/services/v2/file.ts

```diff
@@ -80,6 +80,9 @@ export async function getFilesByModel(user: UserDoc, modelId: string) {
 
 export async function getFilesByIds(user: UserDoc, modelId: string, fileIds: string[]) {
   const model = await getModelById(user, modelId)
+  if (fileIds.length === 0) {
```

Comment:
If you're going to add edge case checks like this, they should probably be at the top of the function.  Did you put it down here so that we're still checking model authorisation?

Reply:
Yes.